cmd/algorand-indexer/daemon_test.go

```diff
@@ -1,9 +1,16 @@
 package main
 
 import (
+	"bytes"
 	"context"
 	"errors"
+	"fmt"
+	"github.com/spf13/pflag"
```

Comment:
Import order. It isn't super strict, but we usually have 4 groups:
1. builtins
2. external deps
3. algorand external deps
4. project deps